opus/application/static_media/js/browse.js

```diff
@@ -728,7 +810,7 @@ var o_browse = {
             // update the data table w/the new columns
             if (!o_utils.areObjectsEqual(opus.prefs.cols, currentSelectedMetadata)) {
                 o_browse.resetData();
-                o_browse.initTable(opus.col_labels);
+                o_browse.initTable(opus.colLabels, opus.colLabelsNoUnits);
```

Comment:
it bothers me that we now have two global vars that are almost identical except a bit of text removed - i would rather see the colLabelsNoUnits become a function; if need be for timely sake you can return/set that global til we find a better way but I see these getting out of sync pretty easily.

Reply:
You can't make it a function. That's the whole point. See the discussion Dave and I had above. Only the Django knows what the two text strings should be. He tried to make the JS compute one from the other, and you just can't do it in a portable way that will stand the test of time.

Reply:
I do agree this whole thing is kind of ugly. This is some of the code I was mentioning earlier today that is going to have to be totally rewritten once we allow multiple search terms, so I just want to wait until then to clean it up because I don't know how we will eventually handle it.